content/admin/github-actions/enabling-github-actions-for-github-enterprise-server/enabling-github-actions-with-minio-storage.md

```diff
@@ -11,22 +11,24 @@ topics:
   - Infrastructure
   - Storage
 redirect_from:
-  - /admin/github-actions/enabling-github-actions-with-minio-gateway-for-nas-storage
-shortTitle: MinIO Gateway for NAS storage
+  - /admin/github-actions/enabling-github-actions-with-minio-storage
+shortTitle: MinIO storage
 ---
 
 {% data reusables.actions.minio-gateways-removal %}
```

Comment:
If we're removing all references to MinIO Gateways for NAS, should we remove this warning from the article?